packages/hint/docs/user-guide/concepts/configurations.md

```diff
@@ -1,43 +1,44 @@
 # Configurations
 
-A `configuration` is a package that contains a `.hintrc`
-configuration file. This makes it easier and faster for users
-to have `webhint` up and running and it also facilitates sharing
-configurations for different things such as: related hints, URLs
-to ignore, etc. When installing a `configuration`, all its
-dependencies (`hint`s, `connnector`s, `formatter`s, `parser`s)
-should be installed automatically as well.
-
-To use a `configuration`, you have to:
-
-1. After installing `webhint`, install a configuration package. When
-   running `npm create hintrc`, the wizard will list you the official
-   configuration packages but you can search on `npm`. Any package
-   `@hint/configuration-` or `webhint-configuration-` should be a valid
-   candidate.
-2. Once installed, update your `.hintrc` to use it (this step is not
-   needed if you are using the wizard):
+A `configuration` is a way to share `.hintrc` values for different use
+cases, such as related hints, URLs to ignore, shared organization
+standards, etc. When installing a `configuration`, all of its
+dependencies (`hint`s, `connnector`s, `formatter`s, `parser`s) should be
+installed automatically as well.
+
+Conveniently, any configuration you choose when running
+`npm create hintrc` is automatically installed _and_ added to your
+`.hintrc` for you.
+
+To manually use a `configuration`, install any package matching
+`@hint/configuration-`, `webhint-configuration-`, or
+`@namespace/webhint-configuration-`. Once installed, update your
+`.hintrc` to use it by adding it to the `extends` array. Given a package
+called `webhint-configuration-example1` OR
+`@orgname/webhint-configuration-example1`, add the following:
 
 ```json
 {
     "extends": ["configuration1"]
```

Comment:
To clarify, the changes in my PR currently only add support for scoped packages when using the full packages name in the configuration. So `@orgname/webhint-configuration-example1` would require the following configuration:

```json
{
    "extends": ["@orgname/webhint-configuration-example1"]
}
```

Also, for the first-party or non-scoped packages, this would need to be `example1` instead of `configuration1` (or could also be the full package name).
```suggestion
     "extends": ["example1"]
```


As a side note, I looked into supporting the same short syntax for scoped packages, but that proved to be a bit more of an invasive change to our resource loader. Going forward I suspect we may switch to recommending full package names anyway to remove confusion about which part of the name matters for the configuration.

Reply:
Okay I'm gonna rewrite a little, would love your thoughts on options for next changes:

- require all packages to be fully qualified
  - pros: consistency; promotes good habits
  - cons: `@hint/` isn't treated extra special
  - other: obviously doesn't actually change any functionality whatsoever, but could make migration easier in the future if the resource loader is refactored for simplicity
- only fully qualify 3rd party packages
  - pros: makes official hint packages really stand out
  - cons: makes hintrc files harder to read; sorta muddies the waters on how to include

(btw if webhintio/hint#2640 is at any point ready to merge, i'll happily just reopen there)

Reply:
oh i forgot to include the option of

- exactly how it works rn, which, con is kinda confusing as evidenced by the fact that i forgot it was even an option and just to confirm:
  - given `webhint-type-name`, the string is `name`
  - `@hint/type-name` string is `name`
  - `@foo/webhint-type-name` string is `@foo/webhint-type-name`

Reply:
Personally I really like making all packages fully qualified in the examples, but I think we should get more input. I'm going to open a Docs issue to kick off this discussion, but I'm optimistic others will agree with biasing toward consistency (and not having to learn some special format beyond the package name). 

Also, webhintio#2640 should hopefully merge today pending any last minute review feedback so I think we'll want to re-open this there.

Reply:
And yes, your list of how it works right now with the changes from webhintio#2640 is correct. I agree it's confusing.